l10n_es_aeat_mod349/views/mod349_view.xml

```diff
@@ -188,6 +188,12 @@
                 <button name="%(l10n_es_aeat.action_wizard_aeat_export)d" position="attributes">
                     <attribute name="name">%(action_wizard_aeat_mod349_export)d</attribute>
                 </button>
+                <field name="period_type" position="replace">
```

Comment:
¿Y no es  bastante lío tener que modificar esto en el aeat base y tener que
actualizarlo solo por este replace?. De verdad que comparto lo de no usar
replace pero creo que todo es evaluable y se deben ver pros y contras y
como dices es poco probable que dé problemas en este contexto.

El 3 de junio de 2016, 13:11, Pedro M. Baeza <notifications@github.com>
escribió:

> In l10n_es_aeat_mod349/views/mod349_view.xml
> <https://github.com/OCA/l10n-spain/pull/367#discussion_r65691694>:
>
> > @@ -188,6 +188,12 @@
> >                  <button name="%(l10n_es_aeat.action_wizard_aeat_export)d" position="attributes">
> >                      <attribute name="name">%(action_wizard_aeat_mod349_export)d</attribute>
> >                  </button>
> > +                <field name="period_type" position="replace">
>
> Ya, si es bastante improbable, pero utilizando el mismo criterio siempre
> no hay posibilidad de error. Puesto que tenemos también el control del
> "super", podemos poner:
>
> <div name="period_type_container">
>     <label for="period_type">
>     <field name="period_type">
> </div
>
> Y ya meter dentro de ese div el check en este módulo.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OCA/l10n-spain/pull/367/files/df8b8a5d94757ae1a5a4332bf4a9860aec3feef7#r65691694>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKpVCyCObu8QrgTL2Y4xZ5Lm9AniY_qgks5qIAvmgaJpZM4ItbTI>
> .
>



-- 
=========================
Santi Argüeso
Twitter: @santiarg

Web: http://comunitea.com
Twitter: @comutuitea
Tfno: 982814805